src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -300,6 +300,25 @@ public bool ShouldVersionOldDocument(DocumentsOperationContext context, Document
             return true;
         }
 
+        public bool CheckTombstoneConflictStatus(DocumentsOperationContext context, string docId, ChangeVector revisionChangeVector)
```

Comment:
let's rename this to something like `IsExistingTombstoneIsNewer`

Reply:
Done